controller/array_action/ds8k_volume_cache.py

```diff
@@ -0,0 +1,66 @@
+from collections import defaultdict
+from threading import RLock
+
+from controller.common.csi_logger import get_stdout_logger
+
+logger = get_stdout_logger()
+
+
+class volumeCacheByAddress:
+    def __init__(self):
+        logger.debug("creating a new cache")
+        self.volume_cache_by_address = defaultdict(dict)
+        self.cache_lock = RLock()

```

Comment:
try to keep members private by default wherever possible, in every class

Reply:
done